### ISSUE TITLE
Enable Tkinter on Windows ARM

### ISSUE THREAD

**Issue by zooba** [python/cpython#83048]
BPO | [38867](https://bugs.python.org/issue38867)
--- | :---
Nosy | @pfmoore, @tjguk, @zware, @zooba, @E-Paine

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2019-11-20.17:41:21.233>
labels = ['type-feature', 'expert-tkinter', '3.9', 'OS-windows']
title = 'Enable Tkinter on Windows ARM'
updated_at = <Date 2021-10-16.16:44:15.859>
user = 'https://github.com/zooba'
```

bugs.python.org fields:
```python
activity = <Date 2021-10-16.16:44:15.859>
actor = 'epaine'
assignee = 'none'
closed = False
closed_date = None
closer = None
components = ['Tkinter', 'Windows']
creation = <Date 2019-11-20.17:41:21.233>
creator = 'steve.dower'
dependencies = []
files = []
hgrepos = []
issue_num = 38867
keywords = []
message_count = 2.0
messages = ['357093', '404101']
nosy_count = 5.0
nosy_names = ['paul.moore', 'tim.golden', 'zach.ware', 'steve.dower', 'epaine']
pr_nums = []
priority = 'normal'
resolution = None
stage = 'needs patch'
status = 'open'
superseder = None
type = 'enhancement'
url = 'https://bugs.python.org/issue38867'
versions = ['Python 3.9']
```

</p></details>

**Comment by aivarannamaa** [comment]
I found some instructions for building Tcl/Tk for Windows arm: https://linaro.atlassian.net/wiki/spaces/WOAR/pages/28698017811/Tcl+Tk

**Comment by zooba** [comment]
Yes, it works now and will be included in 3.11.
